src/ngraph/pass/pass_config.cpp

```diff
@@ -87,6 +87,11 @@ void pass::PassConfig::set_pass_enable(const string& name, bool enable)
     m_pass_enables[name] = enable;
 }
 
+bool pass::PassConfig::get_pass(const string& name) const
```

Comment:
IIUC, any pass is enabled by default: if it is not in `m_pass_enables` then it is enabled. Then you can fix `get_pass_enable` to return true if the pass is not in the map, and remove `get_pass`.

Reply:
m_pass_enables has passes that maybe disabled by env var.
`m_pass_enables.emplace(split_str[0], parse_string<bool>(split_str[1]))`

Reply:
i guess each backend chooses which passes they enable hardcoded, users cant really pick to enable non-enabled backend passes, will make the change.

Reply:
@nmostafa pass_config is expected to be independent of which passes backend enabled by default. In other words, pass_config role is to report what users configured on the command line env (irrespective if the pass is enabled by default on backend). thoughts?

Reply:
I see. Perhaps changing it to `is_pass_disabled()` would be better, and the condition above should be moved to that function.